.github/workflows/publish-homebrew-lambda.yml

```diff
@@ -0,0 +1,15 @@
+name: publish-homebrew-lambda
+on: push
+      branches:
+        - main
+jobs:
+  example:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Invoke homebrew lambda update workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: homebrew-lambda-update.yml
+          repo: hashicorp/releng-support
+          ref: RELENG-1105-automate-lambda-deploy
```

Comment:
Is this a leftover from testing? Could this point to main instead?

Reply:
Yes! good catch